commerce_coordinator/apps/titan/pipeline.py

```diff
@@ -60,14 +68,39 @@ def run_filter(self, edx_lms_user_id, payment_number=None, **kwargs):  # pylint:
         try:
```

Comment:
_(Referring to line 55, 60-64 above)_ Could you add a mention of what checks are performed and what arguments are required for which checks in the docstrings above?

Reply:
Added details.